app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command, file_out, file_in = command.split()
```

Comment:
Check that split_command have exactly 3 parameters

Reply:
before this line

Reply:
command must have 3 arguments, because code will be broken if will be less then 3 here.  check like len(command) == 3 before